Neos.ContentRepository.Core/Classes/Feature/NodeMove/Event/NodeAggregateWasMoved.php

```diff
@@ -63,6 +63,10 @@ public function __construct(
         public ContentStreamId $contentStreamId,
         public NodeAggregateId $nodeAggregateId,
         public ?NodeAggregateId $newParentNodeAggregateId,
+        /**
+         * @var DimensionSpacePoint $dimensionSpacePoint This is one of the *covered* dimension space points of the node aggregate and not necessarily one of the occupied ones. This allows us to move virtual specializations only when using the scatter strategy
+         */
+        public DimensionSpacePoint $dimensionSpacePoint,
```

Comment:
so that we need the originally attempted dimensionSpacePoint to be moved seems correct. But the question is, see https://github.com/neos/neos-development-collection/issues/5360 if also the relationDistributionStrategy should be part of the event?

Whas is your opinion? @nezaniel

Reply:
I think it wouldn't hurt to have the strategy in here if only for pontential debugging help....